app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -214,7 +214,7 @@ class PipelineSampleReport extends React.Component {
     });
     let params = `?${window.location.search.replace("?", "")}&report_ts=${
       this.report_ts
-    }&version=${this.gitVersion}`;
+    }&git_version=${this.gitVersion}&format=json`;
```

Comment:
Why is format sent as param?
Should be in the endpoitn i.e. `report_info.json?...`

Reply:
format is sent because the endpoint is not currently requested as `report_info.json`, and it was necessary to make the cache response header correct. I don't know why `report_info.json`  is not in use. I could fix that.